examples/mcp_servers/authorization/pyproject.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 description = "MCP Server created with Arcade.dev"
 requires-python = ">=3.10"
 dependencies = [
-    "arcade-mcp-server>=1.8.0,<2.0.0",
+    "arcade-mcp-server>=1.12.0,<2.0.0",
```

Comment:
this one is unrelated, but sneaking it into this PR because securing your MCP server is only available in `arcade-mcp-server>=1.12.0`